lib/Runtime/PlatformAgnostic/Intl.h

```diff
@@ -5,6 +5,48 @@
 
 #pragma once
 
+namespace PlatformAgnostic
+{
+namespace Intl
+{
+    // NOTE(jahorto): Keep these enums in sync with those by the same name in Intl.js
+    enum class NumberFormatStyle
+    {
+        DEFAULT = 0, // "decimal" is the default
+        DECIMAL = 0, // Intl.NumberFormat(locale, { style: "decimal" }); // aka in our code as "number"
+        PERCENT = 1, // Intl.NumberFormat(locale, { style: "percent" });
+        CURRENCY = 2, // Intl.NumberFormat(locale, { style: "currency", ... });
+        MAX
+    };
+    enum class NumberFormatCurrencyDisplay
+    {
+        DEFAULT = 0, // "symbol" is the default
+        SYMBOL = 0, // Intl.NumberFormat(locale, { style: "currency", currencyDisplay: "symbol" }); // e.g. "$" or "US$" depeding on locale
```

Comment:
I think it would be more clear if the `DEFAULT` value is listed below so it's obvious which value the default is the same as. I mean like:

```c++
enum class NumberFormatCurrencyDisplay
{
    SYMBOL = 0,
    DEFAULT = SYMBOL,
    ...
}
```

Then if you reorder the values, it'll be easier to make sure `DEFAULT` remains the same as `SYMBOL`.

Reply:
Good call, I can make that change. @dilijev thoughts? (He wrote these enums in particular, I only moved them)

Reply:
Agreed.